modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -22,31 +22,24 @@ namespace LORIS\create_timepoint;
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris-Trunk
  */
-
 class Create_Timepoint extends \NDB_Form
 {
-    /**
-     * This timepoint's candidate
-     *
-     * @var \Candidate
-     */
-    protected $candidate;
 
-    /**
-     * The subprojectID being created
-     */
-    var $subprojectID;
+    public $skipTemplate = true; // stops from looking for a smarty template
+    public $AjaxModule   = true;
+    protected $candidate;
 
     /**
-     * Check user permissions
+     * Determine whether the user has permission to view this page
      *
      * @param \User $user The user whose access is being checked
      *
      * @return bool
+     * @throws \LorisException
      */
     function _hasAccess(\User $user) : bool
     {
-        $candidate =& \Candidate::singleton($this->identifier);
+        $candidate = \Candidate::singleton($this->identifier);
```

Comment:
Just since you're editing this file anyway can you also add the `true` param to in_array below?

Reply:
@johnsaigle when I add true I'm unable to access the page when using my admin account. Any idea why? I'll try debugging in the meantime.